aegis.sh

```diff
@@ -21,7 +21,8 @@ if [ $FVERSION -ge 10 ]; then
    echo "Fortran version $FVERSION"
    SPECIALFFLAGS=1
 fi
-cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT       \
+cmake $SOURCEDIR -DCMAKE_CXX_FLAGS=-isystem\ "/opt/homebrew/include" \
```

Comment:
What are you trying to achieve here?

Reply:
This was needed to find the system dependencies for nlohmann/json. Not sure if this is needed in general, but I added it here for now.

Reply:
If this is some "out-of-band" external, we should probably have a separate recipe for it.